README.md

```diff
@@ -1,13 +1,29 @@
 # Article Linking
 ![Python application](https://github.com/georgetown-cset/article-linking/workflows/Python%20application/badge.svg)
 
-This repository contains a description and supporting code for CSET's current method of
-cross-dataset article linking. Note that we use "article" very loosely, although in a way that to our knowledge
-is fairly consistent across corpora. Books, for example, are included.
+At CSET, we aim to produce a more comprehensive set of scholarly literature by ingesting multiple sources and then
+deduplicating articles. This repository contains CSET's current method of cross-dataset article linking. Note that we
+use "article" very loosely, although in a way that to our knowledge is fairly consistent across corpora. Books, for
+example, are included. We currently include articles from arXiv, Web of Science, Papers With Code, Semantic Scholar,
+The Lens, and OpenAlex. Some of these sources are largely duplicative (e.g. arXiv is well covered by other corpora)
+but are included to aid in linking to additional metadata (e.g. arXiv fulltext).
 
-For each article in arXiv, WOS, Papers With Code, Semantic Scholar, The Lens, and OpenAlex
-we normalized titles, abstracts, and author last names. For the purpose of matching, we filtered out
-titles, abstracts, and DOIs that occurred more than 10 times in the corpus. We then considered each group of articles
+For more information about the overall merged academic corpus, which is produced using several data pipelines including
+article linkage, see the [ETO documentation](https://eto.tech/dataset-docs/mac/).
+
+## Matching articles
+
+To match articles, we need to extract the data that we want to use in matching and put it in a consistent format. The
+SQL queries specified in the `sequences/generate_{dataset}_data.tsv` files are run in the order they appear in those
+files. For OpenAlex we exclude documents with a `type` of Dataset, Peer Review, or Grant. Additionally, we take every
+combination of the Web of Science titles, abstracts, and pubyear so that a match on any of these combinations will
+result in a match on the shared WOS id. Finally, for Semantic Scholar, we exclude any documents that have a non-null
+publication type that is one of Dataset, Editorial, LettersAndComments, News, or Review.
```

Comment:
Are all entries from arXiv & PWC taken without a pub type restriction? Reads a bit weird to specify the steps for OA, WOS, and SemScholar, but not arXiv & PWC.

Reply:
Yes, they're taken without restriction